examples/python_unet/README.md

```diff
@@ -0,0 +1,9 @@
+# U-Net Image Segmentation Inference with AMD MIGraphX
+
+This examples provides a simple example for utilizing U-Net ONNX model for image segmentation, using AMD MIGraphX graph optimization engine for fast inference.
+
+## How-to
+Please utilize the notebook given `unet_inference.ipynb`.
+
+## Model Details
+ONNX model utilized in this example can be found [here](https://github.com/cagery/unet-onnx).
```

Comment:
this should be updated with new model link.

Reply:
done thanks!